sdk/src/user.ts

```diff
@@ -318,7 +318,8 @@ export class User {
 	 */
 	public getPerpPositionWithLPSettle(
 		marketIndex: number,
-		originalPosition?: PerpPosition
+		originalPosition?: PerpPosition,
+		withWeightMarginCategory = undefined
```

Comment:
why does `withWeightMarginCategory` cause the dust value to be added? why doesn't it always get added?

Reply:
think if you dont use a weighting, it means you just want to see net usd value

so in that case you havent really lost anything unless you decide to remove shares

Reply:
could we change it to just be `burnLpShares` or s.t. and cast it as a boolean? `withWeightMarginCategory` typically means we just weight the usd value, but it's different here